src/index.html

```diff
@@ -16,17 +16,29 @@
       href="style.css"
     />
   </head>
-  <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
 
-    <input
-      type="text"
-      data-qa="hover"
-      placeholder="Try “Los Angeles“"
-    />
+  <body>
+    <form
+      class="searchbar"
+      data-qa="search-form"
```

Comment:
The `data-qa` attribute for the first form should be reviewed. Ensure it matches the task requirements for identifying the search form.